app/models/user/UserService.scala

```diff
@@ -84,7 +84,8 @@ class UserService @Inject()(conf: WkConf,
         passwordInfo,
         isAdmin,
         isSuperUser = false,
-        isDeactivated = !isActive
+        isDeactivated = !isActive,
+        lastTaskTypeId = None
```

Comment:
@fm3 why do I need to specify `lastTaskTypeId` here but not in `InitialDataController` ?

Reply:
Do you get compiler errors if you don’t? I don’t see why it would be a problem to omit it.

Reply:
taking that back, I got the error also for InitialDataController. I suppose this has something to do with the other named parameters